tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+	"compilerOptions": {
+		"outDir": "src",
+		"declaration": true,
+		"esModuleInterop": true,
+		"target": "es2022",
+		"module": "es2022",
```

Comment:
not in this pr, but maybe in future, we should investigate es5 for wider a11y? i'm aware of the ie sunset but i still deal with clients who have users on ie

Reply:
i think further transformations could be done through the bundler? unless you're trying to import directly from a CDN